Common/UnitDefinitions/Force.json

```diff
@@ -70,7 +70,7 @@
         {
           "Culture": "ru-RU",
           "Abbreviations": [ "Н" ],
-          "AbbreviationsWithPrefixes": [ "даН", "кН" ]
+          "AbbreviationsWithPrefixes": [ "мкН", "мН", "даН", "кН", "МН" ]
```

Comment:
I forgot to update these yesterday and it messed up the localization. I added a error in the script file for these situations and also found torque had the same issue.